frontend/public/src/containers/pages/profile/EditProfilePage.js

```diff
@@ -13,7 +13,7 @@ import users, { currentUserSelector } from "../../../lib/queries/users"
 import queries from "../../../lib/queries"
 import EditProfileForm from "../../../components/forms/EditProfileForm"
 
-import type { Response } from "redux-query"
+import type { HttpResponse } from "../../../flow/httpTypes"
```

Comment:
After much futzing with my own installation, was able to confirm. (registration/login isn't something I usually care about since i just work in the front end app, but being able to tell that the registration works requires the connection to edx to actually... work.